run_final.sh

```diff
@@ -0,0 +1,22 @@
+# Define arrays for threads, warps, and matrix sizes
+matrix_sizes=(16 32 64 128 256 512)
+tcsizes=(8 16 32)
+tcnums=(4 8 16 32)
+#lsulanes=(4 16)
+#cores=(32)
+
+
+# Loop through each combination of threads and warps
+for size in "${matrix_sizes[@]}"; do
+    sed -i "s/OPTS ?= -n[0-9]\+/OPTS ?= -n${size}/" ../tests/regression/matmul/Makefile
+    sed -i "s/OPTS ?= -n[0-9]\+/OPTS ?= -n${size}/" tests/regression/matmul/Makefile
```

Comment:
Place the file in the correct directory and remove one of these lines.

Reply:
Also I don't think you need to modify the file, you can just set the environmental variable `OPTS` as appropriate.

Reply:
Fixed.